etc/electron.profile

```diff
@@ -7,6 +7,7 @@ include electron.local
 include globals.local
 
 include disable-common.inc
+include disable-exec.inc
```

Comment:
This didn't already have noexec though, right?

Reply:
Right! Is it possible electron.profile is a bit too restrictive anyway?

Reply:
It could be. I think we should remove `disable-exec.inc` from it and do other changes in separate PR.

Reply:
I'll remove it in a second commit.